tests/Functional/FetchJoinAdminTest.php

```diff
@@ -27,4 +27,28 @@ public function testCountFetchJoined(): void
         self::assertSelectorExists('tr[data-author-id="miguel_de_cervantes"]');
         self::assertSelectorExists('tr[data-author-id="anonymous"]');
     }
+
+    public function testBatchAction(): void
+    {
+        $this->client->request(Request::METHOD_GET, '/admin/tests/app/book/list');
+        $this->client->submitForm('OK', [
+            'all_elements' => true,
+            'action' => 'delete',
+        ]);
+        $this->client->submitForm('Yes, execute');
+
+        self::assertResponseIsSuccessful();
+    }
+
+    public function testBatchActionFetchJoined(): void
+    {
+        $this->client->request(Request::METHOD_GET, '/admin/tests/app/author/list');
+        $this->client->submitForm('OK', [
+            'all_elements' => true,
+            'action' => 'delete',
+        ]);
+        $this->client->submitForm('Yes, execute');
+
+        self::assertResponseIsSuccessful();
```

Comment:
With those 2 batch delete tests we test behavior on both cases. Joined query and normal query. It still missing the json field to make postgres fail.